setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
This doesn’t really work in a world of wheels.

You want something like:

```python
extras_require={
    ':python_version<"3.0"': ["Twisted>=13.2.0"],
    ':python_version>"3.0"': ["Twisted>=15.4.0"],
}
```

I’m not sure what setuptools it needs though, maybe @dstufft can advice?